bin/oscapd-evaluate

```diff
@@ -103,6 +103,9 @@ def cli_scan(args, config):
     scanned_targets = []
     failed_targets = []
 
+    if not args.profile.startswith("xccdf_org.ssgproject.content_profile_"):
+        args.profile = "xccdf_org.ssgproject.content_profile_" + args.profile
```

Comment:
@mpreisler  Yes, it will. Arguments can be changed without any problem in Python.

If the given profile ID doesn't start with "xccdf_org.ssgproject.content_profile_", then this prefix will be added to the beginning of the profile ID.

In other words, if user types only profile name, eg. 'pci-dss', it will be changed to proper profile ID 'xccdf_org.ssgproject.content_profile_pci-dss'.

Example usage:
`oscapd-evaluate scan --profile pci-dss --target docker-image://3874a5cf --output /tmp/output/`

Regarding your second question: In oscapd-evaluate, we don't allow users to use anything else than SSG. Users with custom content don't exist.

How do you suggest to improve this so that it's easy to figure out what this code does?